### ISSUE TITLE
twitter reverse order for post with multiple images attatched for user media page viewing

### ISSUE THREAD

**Issue by ujipo77** [MapoMagpie/eh-view-enhance#56]
If a post have multiple images attached, the viewer will display it in an ascending order,  like 1.1, 1.2, 1.3, 1.4. This is not a problem on itself, but when you go to a user and use the viewer to view media, because media display the post at descending order the viewer and the attached images in ascending order it will go like 4.1, 4.2, 4.3, 4.4, 3.1, 3.2, 3.3, 3.4, 2.1, 2.2 and so on.

This is a problem when you are trying to read a comic that spans across multiple post. The comic pages that will be displayed in a users media page will display something like this, 5, 6, 7, 8 and then 1, 2, 3, 4. 

My suggestion to fix this is to add a button that will reverse the order of  image displayed by a post with multiple images attached so it will follow as a descending order like the user media page.

example of media page:
https://x.com/pcmaniac88/media
(noticed how the comics are out of order)

TL;DR:  Make a button to change the order of a post that have multiple images attached, so reading comics in user media is in order

**Comment by MapoMagpie** [comment]
Added, There is a new option of `Descending Images In Post` in config panel. it's default value is true.

**Comment by ujipo77** [comment]
ok the new feature works, thanks
